fastmigrate/core.py

```diff
@@ -406,16 +404,16 @@ def run_migrations(
     
     # Check if database file exists
     if not db_path.exists():
-        console.print(f"[bold red]Error:[/bold red] Database file does not exist: {db_path}")
-        console.print("The database file must exist before running migrations.")
+        sys.stderr.write(f"Error: Database file does not exist: {db_path}\n")
+        sys.stderr.write("The database file must exist before running migrations.\n")
         return False
     
     try:
         # Ensure this is a managed db
         try:
             create_db(db_path)
         except sqlite3.Error as e:
-            console.print(f"""[bold red]Error:[/bold red] Cannot migrate the db at {db_path}.
+            sys.stderr.write(f"""Error: Cannot migrate the db at {db_path}.
```

Comment:
Do you think it might be better to provide a link to instructions, instead of printing a lot here?

Reply:
I agree the error was saying too much, especially because it was trying to summarize the delicate but crucial issue of how to enroll a database properly. I added a link to the shiny new docs page for that.

I do think it is appropriate to provide some instructions here, because this particular error is a good opportunity to teach someone how to use the command and the system. It's not just to communicate to failure to someone who already understands all that.

So I slimmed down the error message and linked to docs but did not eliminate it entirely.

Resolved via 70bc0d5